pkg/kv/kvserver/rditer/testdata/TestReplicaDataIterator/r3/replicatedOnly/skip/output

```diff
@@ -0,0 +1,18 @@
+echo
```

Comment:
Trying to match this "skip" to the test code, but can't find what it corresponds too. Could you help me find it? Or it's a leftover file after some rename?

Reply:
Removed.